docs/utils.md

```diff
@@ -19,9 +19,9 @@ This component requires the same CouchDB options as other components in this mod
 | `couch.userDbName` | `{String}` | The CouchDB database name containing our user records.  Defaults to `users`. |
 | `couch.userDbUrl`  | `{String}` | The URL of our CouchDB instance.  By default this is expanded from `userDbName` above using the pattern `http://admin:admin@localhost:%port/%userDbName` (see above). |
 
-## Component Invokers
+### Component Invokers
 
-## `{that}.createNewUser(userData)`
+### `{that}.createNewUser(userData)`
 
 * Take an object with `username`, `email`, and `password` entries.
 * Returns a promise containing either the new CouchDB record for the account or
```

Comment:
You need to pick apart "containing" better, something like "If account creation is successful, the promise is resolved with X, otherwise the promise is rejected with an error that looks like Y".

Reply:
Done